spec/controllers/review_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+# spec/controllers/reviews_controller_spec.rb
+require "rails_helper"
+
+RSpec.describe ReviewsController, type: :controller do
+  let(:user) { create(:user, role: 0) }
+  let(:tour) { create(:tour) }
+  let(:review) { create(:review, tour: tour, user: user) }
+
+  before do
+    user.confirm
+    sign_in user
+  end
+
+  describe "POST #create" do
+    context "with valid attributes and existing booking" do
```

Comment:
phân biệt describer, context, it

Reply:
describe: Nhóm các test case lại dưới một chủ đề chung. Ví dụ: Testing the functionalities of a User model.

context: Nhóm các test case lại dưới một điều kiện hoặc ngữ cảnh cụ thể. Ví dụ: Testing user validations when creating a new user.

it: Mô tả và định nghĩa một test case cụ thể. Ví dụ: It validates the presence of a username when creating a new user.